lib/ui/window.dart

```diff
@@ -51,18 +51,22 @@ part of dart.ui;
 /// the [viewPadding] anyway, so there is no need to account for
 /// that in the [padding], which is always safe to use for such
 /// calculations.
-///
-/// See also:
-///
-///  * [FlutterWindow], a special case of a [FlutterView] that is represented on
-///    the platform as a separate window which can host other [FlutterView]s.
-abstract class FlutterView {
+class FlutterView {
+  FlutterView._(this._viewId, this.platformDispatcher);
+
+  /// The opaque ID for this view.
+  final Object _viewId;
+  Object get viewId => _viewId;
```

Comment:
The getter that just returns a final private variable can be simplified to:

```suggestion
  /// The opaque ID for this view.
  final Object viewId;
```

Reply:
Alternatively, move the field below the getter and the doc comment should be on the getter.